packages/uni_app/lib/controller/local_storage/database/app_restaurant_database.dart

```diff
@@ -104,7 +111,15 @@ class RestaurantDatabase extends AppDatabase<List<Restaurant>> {
 
   /// Insert restaurant and meals in database
   Future<void> insertRestaurant(Transaction txn, Restaurant restaurant) async {
-    final id = await txn.insert('RESTAURANTS', restaurant.toJson());
```

Comment:
If you see the sentry catch (ask permission to enter), the problem originates here because `Restaurants` table has different attributes than `resturant.toJson()`. Look at the Restaurant Model, you easily see the `meals` property, which is defined as `@JsonKey`.

This problem could be easily fixed if you just remove `meals key` from `restaurant.toJson()` returned object and insert the filtered restaurant to database. As you can see, there is another table called `meals` which will deal with the meals as a one-to-many relationship.

However, since you already started to add meals to restaurants table and we recently implement serialization, we can go further with your solution! 

Delete dead code:
- meals table
- `getRestaurantMeals` method 
- `Restaurant.fromMap()`, where`Restaurant.fromJson()` already done the job 

Make all the adjustments you need to make it work!

Feel free to send me a message on Slack for help